scripts/rhv_tier_analyzer.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
**REQUIRED:** Add a docstring describing this script.   No implementation details just at a high level what it is supposed to do and why.